apps/testingaccessibility/src/hooks/use-nav-state.ts

```diff
@@ -0,0 +1,13 @@
+import {useCurrentAbility} from './use-current-ability'
+
+/**
+ * TODO: consider removing this and using abilities?
```

Comment:
these booleans are generally counter to the idea behind CASL which is more "can" over "is" but for these two particular pieces of state we are are looking at a state machine type structure where we transition from "loading" to "authenticated"

https://xstate.js.org/docs/packages/xstate-fsm/#super-quick-start

introducing a literal state machine is interesting

Reply:
this is only interesting if we think this state is going to become more complex, might also consider just checkoint `state === 'somestate'` in the component and just deleting this altogether

Reply:
tried this out and it was a little silly :)

Reply:
my conclusion here is that for what this is used for it is fine, but if it starts to grow or get used for other areas we night consider making it more "interesting"

this can probably move to the skill-api

Reply:
I agree that this is fine for our current use case, and I like that it's clearly labeled as such. Similarly, I can see the counterarguments and how thinking about it in a more holistic way might lead to some interesting solutions. Already the return statement below becomes a bit weird when I add `canInviteTeam` to it.